src/main/java/seedu/duke/question/Question.java

```diff
@@ -0,0 +1,78 @@
+package seedu.duke.question;
+
+import seedu.duke.HintInterface;
+import seedu.duke.OptionInterface;
+import seedu.duke.OptionListInterface;
+
+import java.util.ArrayList;
+
+public class Question implements QuestionInterface {
```

Comment:
Looks good! Might be good to add some JUnit tests for this class now even if they just assert true for now, just to keep in mind our desired behavior for each method

Reply:
Sure! Working on it!